app/src/main/java/malidaca/marvellisimo/activities/RegisterActivity.kt

```diff
@@ -34,37 +22,21 @@ class RegisterActivity : AppCompatActivity(), View.OnClickListener {
         view = findViewById(android.R.id.content)
         snackbarManager = SnackbarManager()
 
-        database = FirebaseDatabase.getInstance().reference
-        auth = FirebaseAuth.getInstance()
-        mUser = auth.currentUser
     }
 
     override fun onPause() {
-        if (mUser != null)
-            database.child("users").child(mUser!!.uid).child("online").setValue(false)
+        FireBaseService.toggleOnline(false)
         super.onPause()
     }
 
     override fun onResume() {
-        if (mUser != null)
-            database.child("users").child(mUser!!.uid).child("online").setValue(true)
+        FireBaseService.toggleOnline(false)
```

Comment:
Ska det inte vara toggleOnline(true) ?

Reply:
Appen krashar när man startar den, den försöker hämta en användare som inte finns innan man loggat in. Efter att pratat lite så tänkte vi att man inte behöver ha onResume och onPause i Login och Register activity